handsontable.d.ts

```diff
@@ -2041,6 +1988,33 @@ declare namespace Handsontable {
     column: boolean;
     cells: boolean;
   }
+  
+  namespace RecordTranslation {
```

Comment:
Since you plan to move files from `src/translations` to `src/dataMap` (as stated here https://github.com/handsontable/handsontable/pull/5945#discussion_r342056864), I wonder why the TS namespace is called `RecordTranslation`.

I suggest the TS namespace to be consistent with the directory name.

Reply:
The team agreed to move this to a separate PR.